packages/internal-postcss-fix-relative-paths/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import path from 'node:path'
 import type { AtRule, Plugin } from 'postcss'
+import { normalizePath } from './normalize-path'
```

Comment:
I inlined a corrected version of the `normalize-path` NPM package (specifically regarding network shares) and used that method to cleanup the code slightly.